pkg/rootless/rootless.go

```diff
@@ -211,13 +213,24 @@ func createParentOpt(driver portDriver, stateDir string, enableIPv6 bool) (*pare
 	return opt, nil
 }
 
+func trimWhiteSpace(slc []string) []string {
```

Comment:
kinda weird to both in-place modify the list, and also return it, but ok.

Reply:
This should have been part of the standard libraries of Golang, but in the absence of a clear Map function I choose to just include a simple utility method I've found on: https://stackoverflow.com/a/49429437.
This leads me to 2 questions on K3S policy: 
- Does K3S have a standard location of such utility methods?
- Can I use this example, or should I rewrite it myself for copyright purposes?

Reply:
Do we really need this or can we just tell folks to, I don't know, not include unnecessary whitespace in the env var? We don't trim whitespace for any other args or env vars that take a comma-separated list, we just expect people to provide valid input.

Reply:
I'll be happy to remove the trimming, however, this was mostly put in place because people tend to put whitespace around the commas in the CSV string. It would be the difference between:
export K3S_ROOTLESS_COPYUPDIRS="/etc/,/var/run/,/run/"
and:
export K3S_ROOTLESS_COPYUPDIRS="/etc/, /var/run/, /run/"
But  people need to be careful with environment variables and whitespace anyways, so this is not a big issue. I'll prepare the PR in the coming few hours.